variables.tf

```diff
@@ -92,6 +92,12 @@ variable "user_data" {
   type        = string
 }
 
+variable "use_boot_volume_key_as_default" {
+  description = "Set to true to use boot volume encryption key as the default for all volumes."
```

Comment:
This description needs to be clearer. Like what is the impact for a consumer if its set to true VS false ?

Reply:
If you set this value to true, the boot volume key will be used for ALL volumes, including additional attached volumes, and it will override any keys that were specified in the `block_storage_volumes.encryption_key` variable.

"Default" implies to me that it will be used if someone DOESN'T supply a volume key, but it is actually more of an "override"?

(I don't mean for that to be the description, but it needs to convey that somehow)

Reply:
@ocofaigh , @toddgiguere 

I'm thinking of description like this, will it work?

```
variable "use_boot_volume_key_as_default" {
  description = "Set to true to use the boot_volume_encryption_key as default for all volumes, overriding any encryption_key specified. If set to false, the encryption_key provided in `block_storage_volumes.encryption_key` will be used instead."
  type        = bool
  default     = false
}
```